packages/react/src/autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,88 @@
+import { Popover } from "@radix-ui/react-popover";
+import { useCombobox } from "downshift";
+
+import { AutocompleteContextProvider } from "../autocomplete-context";
+import { type BoxProps } from "../box";
+
+type AutocompleteProps = BoxProps<
+  typeof Popover,
+  {
+    disabled?: boolean;
+    initialHighlightedIndex?: number;
+    isItemDisabled?: (item: unknown) => boolean;
+    itemToKey?: (item: unknown) => string;
+    itemToString?: (item: unknown) => string;
+    items: unknown[];
+    onInputValueChange: (inputValue: string) => void;
+    onValueChange?: (value: string) => void;
+    value?: string;
```

Comment:
these should be unknown - generic Item argument
```suggestion
    onValueChange?: (value: Item) => void;
    value?: Item;
```